src/lib/flight_tasks/tasks/Orbit/FlightTaskOrbit.hpp

```diff
@@ -104,6 +105,10 @@ class FlightTaskOrbit : public FlightTaskManualAltitudeSmooth
 	const float _velocity_max = 10.f;
 	const float _acceleration_max = 2.f;
 
+	int _yaw_behaviour =
+		orbit_status_s::ORBIT_YAW_BEHAVIOUR_HOLD_FRONT_TO_CIRCLE_CENTER; /**<  yaw behaviour during the orbit flight according to MAVLink's ORBIT_YAW_BEHAVIOUR enum */
+	float _initial_heading = 0.f; /**< the heading of the drone when the orbit command was issued */
```

Comment:
@dayjaby this is actually the heading of the vehicle when the vehicle command DO_ORBIT gets processed, or when we can say when the `MAV_CMD_DO_ORBIT` is received and processed in the flight controller.